### PR TITLE
(chore) align and clean up dependencies

### PR DESCRIPTION
**Pull request by pirupius**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
- Updates dependencies
- Removes unnecessary dependencies aligns existing ones
- Lint changes

### CODE REVIEW

**Comment by CynthiaKamau** [comment]
Looks like there also some few typescript issues

File: src/components/inputs/multi-select/ohri-multi-select.component.tsx [hunk 1]
```diff
@@ -51,22 +51,22 @@ export const OHRIMultiSelect: React.FC<OHRIFormFieldProps> = ({ question, onChan
     }));
 
   const initiallySelectedQuestionItems = [];
-  questionItems.forEach(item => {
+  questionItems.forEach((item) => {
```
Comment (raghavawasthi24) [inline src/components/inputs/multi-select/ohri-multi-select.component.tsx:54]: Hey, why haven't you use `map` instead of forEach?

Comment (pirupius) [reply src/components/inputs/multi-select/ohri-multi-select.component.tsx:54]: @raghavawasthi24 Thanks for pointing this out. This PR was however for aligning the dependencies and some of the changes to files like this one was due to the linter. There's another PR that has been merged with some fixes. 

Also feel free to take a look at the codebase and suggest improvements, tickets or even PR's where you see fit.

**Comment by pirupius** [comment]
Closed this as it was too stale in favor of https://github.com/openmrs/openmrs-form-engine-lib/pull/176 with a cleaner history

**Pull request closed without merging**